src/matrixroomhandler.ts

```diff
@@ -172,6 +171,15 @@ export class MatrixRoomHandler {
   }
 
   public async ProcessCommand(event: any, context: any) {
+      const intent = this.bridge.getIntent();
+      // Due to #257 we need to check if we are joined.
+      try {
+          await intent.getClient().sendReadReceipt(event.event_id);
```

Comment:
why not check the joinedRooms?

Reply:
Partly because I think doing a sendReciept would be a bit quicker than trying to pull all the joined rooms from synapse and keeping a cache up to date, and partly because now the bot gives feedback when it's read the message albeit fairly pointlessly.